src/dagmc/dagmcmetadata.hpp

```diff
@@ -47,13 +58,20 @@ class dagmcMetaData {
   // finalise the count data
   void finalise_counters();
 
-  std::map<moab::EntityHandle, std::vector<std::string> > get_property_assignments(std::string property,
-      int dimension,
-      std::string delimiters,
-      bool remove_duplicates = true);
+  // Parse property for entities with the specified dimension and delimiters.
+  // Optionally remove duplicate property values if necessary.
+  std::map<moab::EntityHandle, std::vector<std::string>>
+                                                      get_property_assignments(std::string property,
```

Comment:
Not sure why the style guide considers this better...

Reply:
agreed!  @bam241 has suggested that we switch to clang-format.... maybe soon...